CONTRIBUTING.md

```diff
@@ -134,6 +134,19 @@ $ make changelog
 ````
 …and follow the prompts on screen.
 
+### Changelog grammatical mood
+
+Changelog notes are written in the active imperative mood.  They should not end with a period.
```

Comment:
Also, would it be worth including the rule of thumb that Fraser had mentioned? I personally find it helpful. Something like:

> The simple rule is to pretend the message starts with "This change will ..."

Reply:
yeah good idea, I'll add that